src/node_sea.cc

```diff
@@ -59,47 +50,93 @@ SeaFlags operator|=(/* NOLINT (runtime/references) */ SeaFlags& x, SeaFlags y) {
   return x = x | y;
 }
 
-struct SeaResource {
-  SeaFlags flags = SeaFlags::kDefault;
-  std::string_view code;
-  static constexpr size_t kHeaderSize = sizeof(kMagic) + sizeof(SeaFlags);
+class SeaSerializer : public BlobSerializer<SeaSerializer> {
+ public:
+  SeaSerializer() : BlobSerializer<SeaSerializer>(false) {}
```

Comment:
Just a question - Is switching this boolean to `true`, recompiling and running the only way to see the debug logs?

Reply:
Oh I forgot to change this to the `DebugCategory::SEA` instead. Will update that.

Reply:
Ah, `NODE_DEBUG_NATIVE="SEA" <command>` works, great!